### PR TITLE
Deleted some unnecessary files, added more SASS variable support.

### PR DESCRIPTION
**Pull request by Devetec**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,7 +29,8 @@
     "react-router": "^5.2.0",
     "react-router-dom": "^5.2.0",
     "react-scripts": "^3.4.3",
-    "react-scroll": "^1.8.1"
+    "react-scroll": "^1.8.1",
+    "typescript": "^4.2.3"
```
Comment (andrevenancio) [inline package.json]: Remove typescript. 

What is react-scroll? why do we need it?

File: src/components/header/index.js [hunk 2]
```diff
@@ -28,13 +28,13 @@ export const Header = () => {
 
           <div className={styles.right}>
             <Button onClick={context.syncTaquito} secondary>
-              <Secondary>sync</Secondary>
+              <Secondary className={styles.textcolor}>sync</Secondary>
```
Comment (andrevenancio) [inline src/components/header/index.js:31]: I'm not a fan of this API. the styling for the <Secondary /> button should be on the sass file of the button. src/components/button/index.module.scss The idea is to just import components and not worrying about how that variable gets defined.

File: src/components/header/style.module.scss [hunk 3]
```diff
@@ -17,12 +18,17 @@
...
     justify-content: space-between;
+	color: $color;
 
     @include respond-to('desktop') {
       width: 50%;
       margin: 0 auto;
     }
-
+	
+	.textcolor {
```
Comment (andrevenancio) [inline src/components/header/style.module.scss]: You defining this .textcolor but then on the component there's nothing using this class. Its just easier to define the color on the container, and put color: inherit on the css

**Comment by andrevenancio** [comment]
I've added several comments. can you please have a look through them?

File: src/components/header/style.module.scss (hunk 3 above)
Comment (Devetec) [reply src/components/header/style.module.scss]: Will change!

File: package.json (hunk 1 above)
Comment (Devetec) [reply package.json]: I have no idea what react-scroll is, but I didn't add it. To be on the safe side, I don't want to delete it in this PR.

**Comment by Devetec** [comment]
Is this better?

File: src/components/header/index.js (hunk 2 above)
Comment (andrevenancio) [reply src/components/header/index.js:31]: So the idea with <Secondary> is that it takes care of the layout for you. Do not pass classes into it. The button itself reads the variables.

**Comment by andrevenancio** [comment]
Still some comments!

**Review by andrevenancio — APPROVED** [review]

**Pull request merged**